src/Util/GIF.h

```diff
@@ -44,6 +44,7 @@ class GIF {
 	bool nextFrame();
 	Frame getFrame();
 	uint32_t frameDuration();
+	uint32_t getLoopCount() const;
```

Comment:
ovo uvijek vraća 0.
za šta se to koristi? ako sve radi ovako i dok se 0 vrati mi se čini nepotrebno